services/QuillLMS/app/queries/lesson_recommendations_query.rb

```diff
@@ -29,7 +29,7 @@ def recommendations
   end
 
   def get_activities(rec_id)
-    activities = ActiveRecord::Base.connection.execute("SELECT activities.name, activities.uid, activity_classifications.form_url
+    activities = ActiveRecord::Base.connection.execute("SELECT activities.name, activities.id
```

Comment:
I wanted to call out that I appreciate you trimming this query down so that it doesn't pull columns we don't need anymore in additional to adding the new column you need.

That said, do we need the join on `activity_classifications` in this anymore since we don't pull the column anymore?